src/client/platform-utils.js

```diff
@@ -30,11 +30,18 @@ export function clearCache() {
     'sync',
     'wallets'
   ]
-  return Promise.all([RN.AsyncStorage.multiRemove(keys)])
+  return Promise.resolve(RN.AsyncStorage.multiRemove(keys))
```

Comment:
Why do we need to `Promise.resolve` the `multiRemove` call?

```suggestion
  return RN.AsyncStorage.multiRemove(keys)
```